docs/source/get_involved/code_of_conduct.rst

```diff
@@ -28,40 +29,65 @@ conform to the following Code of Conduct.
 1 Introduction
 --------------
 
-sktime is a community-oriented and -led project. We value the
-involvement of everyone in the community. We are committed to creating a
-friendly and respectful place for learning, teaching and contributing.
-All participants in our in-person events and online communications are
-expected to show respect and courtesy to others at all times.
+sktime is a community-oriented and -led project.
+
+We value the involvement of everyone in the community, and strive to be an open
+community. We commit to provide fair and rewarding opportunities for everyone without
+restriction, to participate, learn, and become leaders of the community. We expect from
+members of our community to create such opportunities for others as have been created
+for them.
+
+We are committed to scientific quality and strive to be an open forum on matters
+of technical content and open community governance. All community members are expected
+to adhere to the principle of free speech in academia and science, and, to the best
+of their abilities, contribute to jointly creating
+the best possible science and the best possible open community.
+
+We are committed to creating a friendly and respectful place for learning,
+teaching and contributing. All participants in our in-person events and online
+communications are expected to show respect and courtesy to others at all times.
 
 To make clear what is expected, everyone participating in activities
 associated with the sktime project is required to conform to this Code
 of Conduct. This Code of Conduct applies to all spaces managed by the
 sktime project including, but not limited to, in-person focus groups and
 workshops, and communications online via GitHub.
 
-The lead investigator of the sktime - Dr Franz Király - is responsible
-for enforcing the Code of Conduct. He can be contacted by emailing
-f.kiraly@ucl.ac.uk.
+The Code of Conduct Committee of sktime is responsible for defining and
+enforcing the Code of Conduct, according to Section 4 below.
+
+Incidents can be reported to the Code of Conduct Committee as outlined in Section 3, and
+are handled in accordance with Sections 3 and 4.
+
+The community council of sktime (see current composition on "roles")
+is responsible for ensuring that resolutions of the Code of Conduct committee
+are adhered to, and can be contacted under info@sktime.org.
```

Comment:
what does this mean, its a bit cryptic. Maybe just reformulate?

Reply:
"responsible" as per RACI, technical term.

Do you have a suggestion how to better formulate this?